Utils/hdinsight-node-common/resources/htmlResources/hdinsight/job/html/sparkjobview.js

```diff
@@ -265,7 +265,7 @@ function getSparkDriverLog() {
         executorsObject = JSON.parse(s);
         var hostPort = getDriverPortFromExecutor(executorsObject);
         ipAddress = hostPort.split(":")[0];
-        var url = localhost + projectId + "/jobhistory/logs/" + ipAddress + "/port/30050/" + containerId + "/" + containerId + "/spark/stderr?restType=yarnhistory";
+        var url = localhost + projectId + "/jobhistory/logs/" + ipAddress + "/port/30050/" + containerId + "/" + containerId + "/livy/stderr?restType=yarnhistory";
```

Comment:
Job view didn't have a livy response here. Only active sessions and batches have such response.